.travis.yml

```diff
@@ -0,0 +1,67 @@
+#https://travis-ci.org/IntelPNI/brainiak
+compiler:
+    - gcc
+    - clang
+
+branches:
+  only:
+    - master
+
+matrix:
+  allow_failures:
+    - os: osx
+
+  include:
+    - os: linux
+      dist: trusty
+      sudo: required
+      python: 3.4
+      language: python
+
+    - os: linux
+      dist: trusty
+      sudo: required    
+      python: 3.5
+      language: python
+
+    - os: osx
+      osx_image: xcode7.1
+      sudo: required
+      language: generic
+      env: HINT=Yosemite MPI=openmpi
+      before_install:
+          - brew update
+          - brew install clang-omp
+          - brew install python3 
+
+
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: HINT=ElCapitan MPI=openmpi
+      before_install:
+          - brew update
+          - brew install clang-omp
```

Comment:
Could you please change to `llvm` and change the env vars according to PR #82? This should make it work, so you should also remove the `allow_failures` and the `gcc` attempted workaround.